conf/layer.conf

```diff
@@ -9,7 +9,7 @@ BBFILE_COLLECTIONS += "raspberrypi"
 BBFILE_PATTERN_raspberrypi := "^${LAYERDIR}/"
 BBFILE_PRIORITY_raspberrypi = "9"
 
-LAYERSERIES_COMPAT_raspberrypi = "hardknott honister"
```

Comment:
So in this case it is not an override?

Reply:
re: LAYERSERIES_COMPAT

Reply:
no, here "raspberrypi" is the name of the layer, not OVERRIDE.

Reply:
So a collection is not an override? I always thought it is.